backend/api/grants/types.py

```diff
@@ -26,11 +31,14 @@ class Grant:
     why: str
     notes: str
     travelling_from: str
+    applicant_reply_deadline: Optional[datetime]
+    applicant_message: Optional[str]
```

Comment:
why optional?

Reply:
Deadline will not exists until the grant is approved and sent the email. 
Message can be null if the Applicant didn't reply